src/GoodsList.tsx

```diff
@@ -8,7 +8,11 @@ type Props = {
 export const GoodsList: React.FC<Props> = ({ goods }) => (
   <ul>
     {goods.map(good => (
-      <li key={good.id} data-cy="good">
+      <li
```

Comment:
Use destructuring in line 10.

Reply:
Thanks for the feedback, I've made changes))